File: src/main/java/com/github/eendroroy/sdk/openweather/response/CoOrdResponse.java
package com.github.eendroroy.sdk.openweather.response;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonRootName;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

/**
 * @author indrajit
 */

@ToString
@Setter
@Getter
@JsonRootName("coord")
public class CoOrdResponse implements BaseResponse {
    @JsonProperty("lon")
    public Double lon;

    @JsonProperty("lat")
    public Double lat;
}

File: src/main/kotlin/com/github/eendroroy/sdk/openweathermap/response/CoOrdResponse.kt
package com.github.eendroroy.sdk.openweathermap.response

import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.annotation.JsonRootName

/**
 * @author indrajit
 */
@JsonRootName("coord")
class CoOrdResponse : BaseResponse {
    @JsonProperty("lon")
    var lon: Double? = null

    @JsonProperty("lat")
    var lat: Double? = null
}